sdk/keyvault/azure-security-keyvault-jca/src/main/java/com/azure/security/keyvault/jca/implementation/model/KeyProperties.java

```diff
@@ -16,36 +23,79 @@ public class KeyProperties {
     private String kty;
 
     /**
-     * get key type
-     * @return key type
+     * Get key type
+     *
+     * @return The key type.
      */
     public String getKty() {
         return kty;
     }
 
     /**
-     * set key type
-     * @param kty key type
+     * Set key type.
+     *
+     * @param kty The key type.
      */
     public void setKty(String kty) {
         this.kty = kty;
     }
 
     /**
-     * Is the key exportable.
+     * Indicates whether the key is exportable or not.
      *
-     * @return true if exportable, false otherwise.
+     * @return A value indicating whether the key is exportable or not.
      */
     public boolean isExportable() {
         return exportable;
     }
 
     /**
-     * Set the key to be exportable.
+     * Set a value indicating whether the key is exportable or not.
      *
-     * @param exportable the exportable flag.
+     * @param exportable A value indicating whether the key is exportable or not.
      */
     public void setExportable(boolean exportable) {
         this.exportable = exportable;
     }
+
+    @Override
+    public JsonWriter toJson(JsonWriter jsonWriter) throws IOException {
+        jsonWriter.writeStartObject();
+        jsonWriter.writeBooleanField("exportable", this.exportable);
+        jsonWriter.writeStringField("kty", Objects.toString(this.kty, null));
+
+        return jsonWriter.writeEndObject();
+    }
+
+    /**
+     * Reads an instance of {@link KeyProperties} from the {@link JsonReader}.
+     *
+     * @param jsonReader The {@link JsonReader} being read.
+     *
+     * @return An instance of {@link KeyProperties} if the {@link JsonReader} was pointing to an instance of it, or
+     * {@code null} if it was pointing to JSON {@code null}.
+     *
+     * @throws IOException If an error occurs while reading the {@link KeyProperties}.
+     */
+    public static KeyProperties fromJson(JsonReader jsonReader) throws IOException {
+        return jsonReader.readObject(reader -> {
+            KeyProperties deserializedKeyProperties = new KeyProperties();
+
+            while (reader.nextToken() != JsonToken.END_OBJECT) {
+                String fieldName = reader.getFieldName();
+
+                reader.nextToken();
+
+                if ("exportable".equals(fieldName)) {
+                    deserializedKeyProperties.exportable = reader.getNullable(JsonReader::getBoolean);
```

Comment:
Don't need to use `getNullable` for either deserialization calls. If the JSON property `exportable` is `null` then this class has an invalid definition. `getString` always supports returning `null`, so calling `getNullable` is just extra overhead that doesn't accomplish anything.